syslog-ng/syslog-ng.conf

```diff
@@ -31,12 +31,19 @@ source s_network {
 };
 
 destination d_local {
-	file("/var/log/messages");
-	file("/var/log/messages-kv.log" template("$ISODATE $HOST $(format-welf --scope all-nv-pairs)\n") frac-digits(3));
+	pipe("/dev/xconsole");
 };
 
+#internal events should log out to console so docker logging driver can find them
 log {
 	source(s_local);
+	destination(d_local);
+};
+
+@include "conf.d/*.conf"
+
+#internal events should log out to console so docker logging driver can find them
```

Comment:
I would prefer naming the destinations, instead of comments. Instead of d_local, maybe d_docker_logging?